### PR TITLE
UI

### PR DESCRIPTION
**Pull request by Olamarx**
- Addded the App file
 - Added the main file
 - Makes it look like the video given

### CODE REVIEW

File: methods.rb [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    case person
+    when 1
+      data = create_student
+      student = Student.new(classroom: nil, age: data[:age], name: data[:name],
+                            parent_permission: data[:permission])
+      puts 'Student was successfully created!'
+      student
+
+    when 3
+      data = create_teacher
```
Comment (ahmetbozaci) [inline methods.rb]: - [ ] Kindly change line 34 to 'when 2' otherwise it cant create a teacher when pressing 2
image

File: methods.rb [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    date = gets.chomp
+
+    if (book <= books.length && book >= 0) && (person <= people.length && person >= 0)
+      rental = Rental.new(date, books[book], people[person])
+      puts '> Rental successfully created'
+      rental
+    else
+      puts '> Invalid option'
+    end
+  end
```
Comment (ahmetbozaci) [inline methods.rb:94]: - [ ] I can't create a rental. Kindly fix it.

image
**_I cant test the list all rental work or not. Next code reviewer kindly check it._**

File: methods.rb [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    end
+    puts '> No books added for the moment' if books.empty?
+  end
+
+  def list_all_people(people)
+    people.each_with_index do |person, index|
+      put "#{index}) [#{person.class.name}] Name: #{person.name} ID: #{person.id} Age: #{person.age}"
+    end
+    puts '> No people added for the moment' if people.empty?
+  end
```
Comment (ahmetbozaci) [inline methods.rb]: - [x] I cant list all people. Kindly fix it.

image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @Olamarx  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: methods.rb (hunk 3 above)
Comment (Olamarx) [reply methods.rb]: @ahmetbozaci thank you, i have fixed this, I forgot to put add a to the put 🙈

File: methods.rb (hunk 1 above)
Comment (Olamarx) [reply methods.rb]: Oh, a mistake. Thank you, I have fixed it.

File: methods.rb (hunk 2 above)
Comment (Olamarx) [reply methods.rb:94]: Thank you, all issue resolved.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Olamarx 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the requested changes from the previous reviewer 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Olamarx** [comment]
@MahmoudBakr23 Thank you very much.

**Pull request merged**